### ISSUE TITLE
Some questions about loss function, dataset, pre-trained model

### ISSUE THREAD

**Issue by kangya998** [WanquanF/NeuralPoints#20]
Hi, feng
Your method has been a great success, I have doubts about some of the key steps.
1. about loss function: is the public code the final version, I observed that the loss function in the code is different from the one described in the paper, especially the integration quanlity loss.
2、about dataset, 13 test models but there is a duplicate model, is this model also used to calculate the average evaluation metrics? The scale of the final generated point cloud is different from the size of the ground truth mesh in PUGeo-Net, how to calculate the p2f distance? Can you share the code of the dataset creation part?
3、about pretrained model, I used pre-trained model for test, how to evaluate the generated points with ground truth 40000. 160000, how to get 10000, 40000, 160000 points for train and test, I tried Can you share the code of the evaluation strategy section? How to use the pre-trained model to get the results in the paper.
I am very much looking forward to your answer, I may have missed some key information.

**Comment by Taousst** [comment]
Hi, are you working on point cloud sampling as well? Can you add a friend to discuss it

**Comment by T-USST** [comment]
> Hi, feng Your method has been a great success, I have doubts about some of the key steps.
> 
> 1. about loss function: is the public code the final version, I observed that the loss function in the code is different from the one described in the paper, especially the integration quanlity loss.
>    2、about dataset, 13 test models but there is a duplicate model, is this model also used to calculate the average evaluation metrics? The scale of the final generated point cloud is different from the size of the ground truth mesh in PUGeo-Net, how to calculate the p2f distance? Can you share the code of the dataset creation part?
>    3、about pretrained model, I used pre-trained model for test, how to evaluate the generated points with ground truth 40000. 160000, how to get 10000, 40000, 160000 points for train and test, I tried Can you share the code of the evaluation strategy section? How to use the pre-trained model to get the results in the paper.
>    I am very much looking forward to your answer, I may have missed some key information.

Hi, are you working on point cloud sampling as well? Can you add a friend to discuss it
